src/tiles/list/tile-list.ts

```diff
@@ -331,6 +331,23 @@ export class TileList extends LitElement {
     >`;
   }
 
+  /** The URL of this item's mediatype collection, if defined. */
+  private get mediatypeURL(): string | typeof nothing {
+    if (!this.baseNavigationUrl || !this.model?.mediatype) return nothing;
+
+    // Need special handling for certain mediatypes that don't have a top-level collection page
+    switch (this.model.mediatype) {
+      case 'collection':
+        return `${this.baseNavigationUrl}/search?query=mediatype:collection&sort=-downloads`;
```

Comment:
TIL `sort=-downloads`

Reply:
I think the home page uses this strategy too for the collection icon in the hero block (list most viewed collections). 🙂